lib/reporters/cli/index.js

```diff
@@ -249,13 +258,12 @@ PostmanCLIReporter = function (emitter, reporterOptions, options) {
                 `${resHeadersLen}${colors.gray(symbols.down)} ${colors.gray('headers')}`,
             `${_.get(res, 'cookies.members.length')} ${colors.gray('cookies')}`
         ].join(` ${colors.gray(symbols.star)} `));
-
         // print request body
         if (reqTextLen) {
-            // truncate very large request (is 2048 large enough?)
-            if (reqTextLen > BODY_CLIP_SIZE) {
-                reqText = reqText.substr(0, BODY_CLIP_SIZE) +
-                    colors.brightWhite(`\n(showing ${util.filesize(BODY_CLIP_SIZE)}/${util.filesize(reqTextLen)})`);
+            // truncate very large request (if noVerboseOutputLimit flag is set, remove any limitations)
+            if (!reporterOptions.noVerboseOutputLimit && reqTextLen > bodyClipSize) {
```

Comment:
If the above recommendations are followed, then this if block become almost as original.